src/utils/emoji.js

```diff
@@ -4,3 +4,4 @@ const supportsEmoji = process.platform !== 'win32' || process.env.VSCODE_PID;
 exports.progress = supportsEmoji ? '⏳' : '∞';
 exports.success = supportsEmoji ? '✨' : '√';
 exports.error = supportsEmoji ? '🚨' : '×';
+exports.warning = supportsEmoji ? '⚠️' : '⚠';
```

Comment:
The provided Windows fallback `'⚠'` doesn't actually work on Windows and will render as `[]`. Please use `‼` (203C).

Reply:
Thanks, I pushed a new commit with both issues fixed.